query_and_submit.py

```diff
@@ -21,19 +21,49 @@
 metrics = {}
 
 for metric_name, query in config["queries"].items():
-  sql = query["countSql"]
-  condition_keyword = "and" if re.search("where", sql, re.IGNORECASE) else "where"
+  count_sql = query.get("countSql")
+  content_sql = query.get("contentSql")
   timestamp_col = query["timestampColumn"]
-  metrics[metric_name] = json.loads(subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]))
-
-requests.post(
-  url=config["endpoint"],
-  json={
-    "startInterval": interval_start,
-    "endInterval": interval_end,
-    **metrics,
-  },
-  headers={
-    "xc-token": metrics_token,
-  }
-)
+  
+  if content_sql:
+    condition_keyword = "and" if re.search("where", content_sql, re.IGNORECASE) else "where"
+    db_result = subprocess.check_output(["psql", os.environ["DBURI"], "-Atc", f"{content_sql} {condition_keyword} \"{timestamp_col}\" between '{interval_start}' and '{interval_end}'"]).decode().split("\n")
+    db_result = filter(lambda x: x != "", db_result)
+
+    content_metric = {}
+    for result in db_result:
+      category, content, created_at = result.split("|")
+
+      content_metric["category"] = category
+      content_metric["content"] = content
+      content_metric["date"] = created_at
+
+      json_output = {
+        "createdAt": interval_start,
+        "endInterval": interval_end,
```

Comment:
I am wondering whether we still need those fields? Are those columns used in the table?